src/App.jsx

```diff
@@ -19,29 +19,28 @@ class App extends Component {
         <nav className="navbar navbar-default navbar-fixed-top navbar-expand{-sm|-md|-lg|-xl} navbar-expand-lg navbar-light bg-light">
           <a href="/" className="navbar-brand">
             <img src={logo} width="30" className="d-inline-block align-top" alt=""></img>
-            &nbsp; SE Soc
           </a>
           <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
             <span className="navbar-toggler-icon"></span>
           </button>
           <div className="collapse navbar-collapse primary-header" id="navbarSupportedContent">
             <div className="nav navbar-nav mr-auto">
-              <a activeClassName="active" className="nav-link nav-item" href="/">Home</a>
-              <a activeClassName="active" className="nav-link nav-item" href="/about" >About Us</a>
-              <a activeClassName="active" className="nav-link nav-item" href="/contact">Contact Us</a>
-              <a activeClassName="active" className="nav-link nav-item" href="/events">Events</a>
-              <a activeClassName="active" className="nav-link nav-item" href="/council">Council</a>
+              <a activeClassName="active" className="nav-link nav-item" href="/">home</a>
+              <a activeClassName="active" className="nav-link nav-item" href="/events">events</a>
+              <a activeClassName="active" className="nav-link nav-item" href="/news">news</a>
+              <a activeClassName="active" className="nav-link nav-item" href="/council">council</a>
+              <a activeClassName="active" className="nav-link nav-item" href="/about" >about</a>
             </div>
           </div>
         </nav>
         <Router>
           <Switch>
             <Route exact path='/' component={Home}/>
-            <Route exact path='/platforms' component={Platforms}/>
             <Route exact path='/events' component={Events} />
-            <Route exact path='/about' component={About} />
-            <Route exact path='/contact' component={Contact} />
+            <Route exact path='/news' component={News} />
             <Route exact path='/council' component={Council} />
+            <Route exact path='/about' component={About} />
+            <Route exact path='/platforms' component={Platforms}/>
```

Comment:
Why the change in ordering here?

Reply:
Platforms is hidden in the navbar and I ordered the rest of the routes in navbar order.